ui/src/dashboards/components/TemplateVariableTable.js

```diff
@@ -0,0 +1,77 @@
+import React, {PropTypes} from 'react'
+
+import TemplateVariableRow from 'src/dashboards/components/TemplateVariableRow'
+import EmptyRow from 'src/admin/components/EmptyRow'
+
+const TemplateVariableTable = ({
+  source,
+  templates,
+  onRunQuerySuccess,
+  onRunQueryFailure,
+  onDelete,
+  tempVarAlreadyExists,
+}) => (
+  <div className="table-custom">
+    {templates.length
+      ? <div>
```

Comment:
What's the context behind styling these things as tables? And if we're styling them as tables, why not use actual table elements?